frontend/src/components/ProjectCard/projectCardStyle.ts

```diff
@@ -11,7 +11,7 @@ export const CardContainer = styled('button')`
   color: inherit;
   cursor: pointer;
 
-  &:hover, &:focus {
+  &:hover${props => !(props.istemplate === 'true') && ', &:focus'} {
```

Comment:
Was there an issue with a boolean?

Reply:
Yes 😢 This is what I got when trying to pass in istemplate={true}
<img width="766" alt="Screen Shot 2023-05-29 at 1 53 23 pm" src="https://github.com/automatarium/automatarium/assets/79517325/b33a514b-c9f0-42f9-95cc-b76f8b5960b8">

Reply:
Ah I know the problem

Put `$` before the name so it becomes `$isTemplate`. Goober is setup to not pass properties that are prefixed with `$` to the DOM so we can use them for pure logic (least I think thats what it does)

Reply:
Memory is a bit hazy but I thought Goober was set up to _prefer_ transitive ($) properties? As always the expert is Benji or Ewan :)

Reply:
```ts
  // Remove transient props from the DOM
  props => Object.keys(props).forEach(p => p[0] === '$' && delete props[p])
```
Seems to remove it, although I'm unsure in my answer now since I can't seem to reproduce the warning? So don't know if doing that will actually fix it

Reply:
Ok was able to finally reproduce :rocket:
Suggestion works, changing to `$isTemplate` and updating `Templates.tsx` to use the new property (which I wasn't doing when checking which is why I got no error) allows to use booleans instead

Reply:
Thanks for getting to the bottom of this! Will update it